### ISSUE TITLE
Entity.GetModel doesn't return real virtual fs path

### ISSUE THREAD

**Issue by Noiwex** [Facepunch/garrysmod-issues#2246]
Yeah, I know that it only happens with stupidly compiled models, but relying on model's hardcoded path isn't right?

See: http://steamcommunity.com/sharedfiles/filedetails/?id=482847417

**Comment by robotboy655** [comment]
I don't think it is supposed to return the "real virtual fs path", it is supposed to return whatever the model's name is, which is not a file path at all for brushes for example.

**Comment by Noiwex** [comment]
oh fie, can we get real model path then?

**Comment by Noiwex** [comment]
I guess best way is not to have models that have been renamed after compilation, since models have their filepath inside themselves.

**Comment by polivilas** [comment]
As a workaround you can make a translation table for badly placed models
PoC:
https://gist.github.com/polivilas/61e188a1666b2c00002d549d38ecbab5
